package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "pretest": "node pretest.js",
-    "test": "mocha --timeout 20000 --require test/init.js",
+    "test": "mocha --timeout 60000 --require test/init.js test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
```

Comment:
Note: for whoever working on any similar tasks, adding timeout/making it longer may fix timing-related errors.

Reply:
Ideally, we should move the timeout configuration to `test/mocha.opts` or a mocha-rc file, so that we can run individual test files via `npx mocha node_modules/juggler-v3/test.js` and don't have to specify the timeout in CLI args.